Packs/CTIX/Integrations/CTIXv3/CTIXv3.yml

```diff
@@ -7,1141 +7,1257 @@ display: CTIX v3
 category: Data Enrichment & Threat Intelligence
 description: This is Cyware Threat Intelligence eXhange(CTIX) integration which enriches IP/Domain/URL/File Data.
 configuration:
-- display: Endpoint URL
-  name: base_url
-  type: 0
-  required: true
-  additionalinfo: Enter the endpoint URL of your CTIX Instance.
-- display: Access Key
-  name: access_id
-  type: 4
-  required: true
-  additionalinfo: Enter the Access Key from the CTIX application.
-- display: Secret Key
-  name: secret_key
-  type: 4
-  required: true
-  additionalinfo: Enter the Secret Key from the CTIX application.
-- display: Trust any certificate (not secure)
-  name: insecure
-  type: 8
-  required: false
-- display: Use system proxy settings
-  name: proxy
-  type: 8
-  required: false
+  - display: Endpoint URL
+    name: base_url
+    type: 0
+    required: true
+    additionalinfo: Enter the endpoint URL of your CTIX Instance.
+  - display: Access Key
+    name: access_id
+    type: 4
+    required: true
+    additionalinfo: Enter the Access Key from the CTIX application.
+  - display: Secret Key
+    name: secret_key
+    type: 4
+    required: true
+    additionalinfo: Enter the Secret Key from the CTIX application.
+  - display: Trust any certificate (not secure)
+    name: insecure
+    type: 8
+    required: false
+  - display: Use system proxy settings
+    name: proxy
+    type: 8
+    required: false
+  - additionalinfo: Reliability of the source providing the intelligence data.
+    defaultvalue: C - Fairly reliable
+    display: Source Reliability
+    name: integrationReliability
```

Comment:
While this was add to the YML, I do not see any use of it in the code itself, when creating the DBotScore.

Reply:
Removed for now. I added this because of a validation error that I ran into.

Reply:
Usually we just pass it when we create the DBotScore.